libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -112,3 +114,38 @@ void AP_GPS_Backend::fill_3d_velocity(void)
     state.velocity.z = 0;
     state.have_vertical_velocity = false;
 }
+
+
+void AP_GPS_Backend::_detection_message(char *buffer, const uint8_t buflen) const
+{
+    const uint8_t instance = state.instance;
+    const struct AP_GPS::detect_state dstate = gps.detect_state[instance];
+
+    if (dstate.detect_started_ms > 0) {
```

Comment:
SBF, GSOF and NOVA will have a positive number here. Has it been tested that their baudrate is correct and valid (because it wasn't used before and according to @WickedShell's comments it looks like baud won't be correct)?